ros_performance_test/TestMessageHeader header
uint8[] data
